packages/discord.js/typings/index.test-d.ts

```diff
@@ -2406,7 +2407,7 @@ new ButtonBuilder({
 // @ts-expect-error
 new EmbedBuilder().setColor('abc');
 
-new EmbedBuilder().setColor('#ffffff');
+new EmbedBuilder().setColor(resolveColor('#ffffff'));
```

Comment:
In context of mainlib discord.js this would be a test for the return type of resolveColor, the builder‘s methods get tested in `@discordjs/builders` already.

the failing test above this one is also redundant.